code/script.js

```diff
@@ -1,51 +1,212 @@
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
-
+const chat = document.querySelector("#chat");
+const inputWrapper = document.querySelector("#input-wrapper");
+const nameInput = document.querySelector("#name-input");
+const nameForm = document.querySelector("#name-form");
 // If you need any global variables that you can use across different functions, declare them here:
-
+//let userAnswer;
 
 // Declare your functions after this comment
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
-  if (sender === 'user') {
+  if (sender === "user") {
+    console.log("user som skriver");
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/girldrinking.png" alt="User" />  
       </section>
-    `
+    `;
     // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
+  } else if (sender === "bot") {
+    console.log("bot som skriver");
+
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/cuterobot.jpg" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
-}
+};
 
 // Starts here
+
+//Greeting
 const greeting = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage("Hello and welcome to the Coffee by D&P! What's your name?", "bot");
+  console.log("greeting"); //can be omitted
+};
+
+
+//User answering his name
+const handleInput = (event) => {
+  event.preventDefault(); //prevents refreshing the page
+  let userName = nameInput.value;
+  showMessage(`Hi! I am ${userName}.`, "user"); //calling on function
```

Comment:
Would be nice if the name input was required. Now you can move on without entering a name.